src/bindings/python/README.md

```diff
@@ -0,0 +1,56 @@
+# OpenVINO Python API
+
+OpenVINO Python API is a part of OpenVINO library. The component is responsible for:
+
+* Bindings of OpenVINO - allowing users to utilize OpenVINO library in their Python code. Python API provides bindings to basic and advanced APIs from OpenVINO Runtime.
+* Extending OpenVINO with pythonic features - on top of direct translations from C++, Python API component:
+    * Adds specific extensions to support numpy-based data.
+    * Provides support for external frameworks inputs.
+    * Provides shortcuts and helpers with more pythonic design.
+    * Allows to apply advanced concepts like shared memory to take full advantage of OpenVINO.
+
+## Key contacts
+
+If you have any questions, feature requests or want us to review your PRs, send us a message or ping us on Github.
+
+Team members and main responsibilities:
+
+* [Jan Iwaszkiewicz](https://github.com/jiwaszki) (e-mail: jan.iwaszkiewicz@intel.com)  
+Techincal Lead: questions, issues and feature requests, memory and GIL related questions
+* [Anastasia Kuporosova](https://github.com/akuporos) (e-mail: anastasia.kuporosova@intel.com)  
+Developer: building of the component, general bindings
+* [Przemyslaw Wysocki](https://github.com/p-wysocki) (e-mail: przemyslaw.wysocki@intel.com)  
+Developer: project requirements, general bindings
+
+## Components
+
+```
+openvino/src/bindings/python
+├── docs                    <-- Developer documentation
+│   └── examples            <-- Developer code examples
+├── README.md               <-- This file:)
+├── src
+│   ├── compatibility       <-- Sources for compatibility API
+│   ├── openvino            <-- Python sources for current API
+│   └── pyopenvino          <-- C++ sources for current API
+├── tests
+├── tests_compatibility
+├── thirdparty              <-- Thirdparty libraries
+│   └── pybind11
+└── wheel                   <-- Wheel specific directory
+```
+
+## Tutorials
+
+If you want to contribute to OpenVINO Python API, here is the list of learning materials and project's guidelines:
+
+* [How to contribute](docs/contributing.md)
```

Comment:
Do we have a plan to change and review these files?
At the current moment it looks strange:
<img width="1158" alt="Screenshot 2022-11-18 at 16 24 46" src="https://user-images.githubusercontent.com/4737252/202704491-776ba088-0511-4a9c-a022-2308942e51d2.png">